### PR TITLE
Completed javascript-primitives-and-variables

### PR DESCRIPTION
**Pull request by JimmyTran1206**
image

### CODE REVIEW

File: javascript-primitives-and-variables/main.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+console.log('full name: ', fullName);
+console.log('type of full name: ', typeof fullName);
+console.log('is kool? ', isCool);
+console.log('type of isCool: ', typeof isCool);
+console.log('total pets: ', totalPets);
+console.log('type of totalPets: ', typeof totalPets);
+const unknown = null;
+console.log('unknown variable: ', unknown);
+console.log('type of unknown: ', typeof unknown);
+const never = undefined;
```
Comment (shawnkost) [inline javascript-primitives-and-variables/main.js:13]: This should be using the `let` keyword, and you shouldn't be assigning any value to this variable.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change here Jimmy!

File: javascript-primitives-and-variables/main.js (hunk 1 above)
Comment (JimmyTran1206) [reply javascript-primitives-and-variables/main.js]: Got it. I have corrected the code: 
image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**